pyproject.toml

```diff
@@ -34,13 +34,13 @@ plugins = ["musicbrainzngs", "pyinotify", "dbus-python", "paho-mqtt", "soco", "p
 [tool.poetry.dev-dependencies]
 pytest = '^7.1.1'
 polib = '^1.1.0'
-sphinx = '^4.4.0'
-sphinx_rtd_theme = '*'
+sphinx = '^5.0.2'
+sphinx_rtd_theme = '^1.0.0'
 sphinx-autobuild = "^2021.3.14"
 xvfbwrapper = { version = "^0.2", platform = "linux" }
-flake8 = "^3.9.2"
+flake8 = "^4.0.1"
```

Comment:
If we change this _here_... it now appears in a bunch of places too, perhaps it should be consistent

Reply:
done